engine/src/witness/btc_e.rs

```diff
@@ -130,12 +132,19 @@ impl VoterApi<BitcoinBlockHeightTracking> for BitcoinBlockHeightTrackingVoter {
 
 		let best_block_header = header_from_btc_header(self.client.best_block_header().await?)?;
 
-		if best_block_header.block_height <= witness_from_index {
-			Err(anyhow::anyhow!("btc: no new blocks found since best block height is {} for witness_from={witness_from_index}", best_block_header.block_height))
-		} else if witness_from_index == 0 {
-			headers.push_back(best_block_header);
-			Ok(InputHeaders(headers))
+		if best_block_header.block_height <= election_property {
+			Err(anyhow::anyhow!("btc: no new blocks found since best block height is {} for witness_from={election_property}", best_block_header.block_height))
 		} else {
+			let witness_from_index = if election_property == 0 {
+				tracing::info!(
+					"bht: election_property=0, best_block_height={}, submitting last 6 blocks.",
+					best_block_header.block_height
+				);
+				best_block_header.block_height.saturating_sub(6)
```

Comment:
Going to be easy to forget this later. Also, we should probably rename `SAFETY_MARGIN` to be something like `MAX_BUFFER`, since the actual margin of safety will be determined by the process_block (or equivalent here)

```suggestion
				 best_block_header.block_height.saturating_sub(BitcoinBlockHeightTrackingTypes::SAFETY_MARGIN)
```